gradio/routes.py

```diff
@@ -144,6 +144,20 @@ def get_blocks(self) -> gradio.Blocks:
             raise ValueError("No Blocks has been configured for this app.")
         return self.blocks
 
+    @staticmethod
+    def build_proxy_request(url_path):
+        url = httpx.URL(url_path)
+        is_hf_url = (
+            url.host.endswith(".huggingface.co")
+            or url.host.endswith(".hf.space")
```

Comment:
Since we just need this to fetch files from upstream spaces, we just need `.hf.space`, right?

Reply:
I think it should be fine to limit to just `.hf.space` yes, I'll make the change